examples/oracle_swap.ts

```diff
@@ -11,12 +11,11 @@ import { OracleBasedCurveCalculator } from "@/gfx/cpmm/curve/oracleCalculator";
 const RPC_URL = process.env.RPC_URL!;
 const SEND_RPC_URL = process.env.SEND_RPC_URL ?? RPC_URL;
 const KEYPAIR_PATH = process.env.KEYPAIR_PATH!;
-const AMOUNT = new BN(parseInt(process.env.AMOUNT!));
+const AMOUNT = new BN(1_000_000);
```

Comment:
We should still leave the option to be set this in the env

```suggestion
const AMOUNT = new BN(process.env.AMOUNT ?? '1_000_000');
```